src/pickers/_datepicker.scss

```diff
@@ -0,0 +1,343 @@
+.mdl-datepicker {
+  * {
+    box-sizing: border-box;
+  }
+
+  &--landscape {
+    .mdl-datepicker__widget {
+      width: 440px;
+    }
+
+    .mdl-datepicker__header {
+      bottom: 0;
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: 160px;
+    }
+
+    .mdl-datepicker__calendar {
+      margin-left: 160px;
+    }
+  }
+
+  &__backdrop {
+    background: $datepicker-backdrop-bg-color;
+    bottom: 0;
+    left: 0;
+    overflow-x: hidden;
+    overflow-y: auto;
+    position: fixed;
+    right: 0;
+    top: 0;
+    z-index: 2147483646;
```

Comment:
Wow, that's a large value! This is to implement the backdrop that leaves everything else obscured, correct? If so, let's use the same value we have in snackbar: 10000. We'll create a z-index file at some point and try to standardise the whole thing, but it should be good enough for now.